tests/test_dsp_resample.py

```diff
@@ -128,3 +130,17 @@ def test_resample_multidimensional_impulse():
     sinc = pf.dsp.time_window(sinc, [N*L/2-int(c/2), N*L/2+int(c/2)],
                               window='hamming')
     np.testing.assert_almost_equal(resampled.time, sinc.time, decimal=2)
+
+
+def test_resample_suppress_aliasing():
```

Comment:
Maybe add test for the filter properties like zero-phase, attenuation?

Reply:
Good point, improved the testing to asses pass- and stop-band properties more precicely